### PR TITLE
Collapsing Survey Pop-up

### PR DESCRIPTION
**Pull request by SMakaiTakori**
## Changes

This PR addresses the need for the OwnPath pop-up survey to be collapsible so users have the option to come back to the survey at their convenience.

## Screenshots

[Collapsible Survey Demo](https://www.loom.com/share/76f1cc3f0e9242639067adb3becc96da)

## Checklist

- ~[ ] if changes filter/search functionality, adds or updates unit tests~
- ~[ ] if adds a new page, adds accessibility tests~
- ~[ ] if adds a new page or new interaction, adds e2e test~
- ~[ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)~
- ~[ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)~

Fixes [Make Survey collapsible Asana ticket](https://app.asana.com/0/1202105544020589/1203872672542969/f)

### CODE REVIEW

**Review by jlhogan — APPROVED** [review]
LGTM, but I think it will be better to let the heading component be configurable

File: src/images/close-survey.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg height="15px" id="Layer_1" style="enable-background:new 0 0 512 512;" version="1.1" viewBox="0 0 512 512" width="15px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M443.6,387.1L312.4,255.4l131.5-130c5.4-5.4,5.4-14.2,0-19.6l-37.4-37.6c-2.6-2.6-6.1-4-9.8-4c-3.7,0-7.2,1.5-9.8,4  L256,197.8L124.9,68.3c-2.6-2.6-6.1-4-9.8-4c-3.7,0-7.2,1.5-9.8,4L68,105.9c-5.4,5.4-5.4,14.2,0,19.6l131.5,130L68.4,387.1  c-2.6,2.6-4.1,6.1-4.1,9.8c0,3.7,1.4,7.2,4.1,9.8l37.4,37.6c2.7,2.7,6.2,4.1,9.8,4.1c3.5,0,7.1-1.3,9.8-4.1L256,313.1l130.7,131.1  c2.7,2.7,6.2,4.1,9.8,4.1c3.5,0,7.1-1.3,9.8-4.1l37.4-37.6c2.6-2.6,4.1-6.1,4.1-9.8C447.7,393.2,446.2,389.7,443.6,387.1z"/></svg>
```
[the remark this answers is not in the document]
Comment (SMakaiTakori) [reply src/images/close-survey.svg:1]: Ah, will do, I didn’t see we had a similar one already. Thanks!

**Comment by alex-cds** [comment]
@jlhogan Looking sweet! Can you please make the x and drowndown arrow the action blue color? (Same color as the x currently) Then we should be good to go.
image

**Comment by SMakaiTakori** [comment]
@alex-cds The color change is up when you have a change to review, thanks!

update-arrow-colors

**Review by alex-cds — APPROVED** [review]
@jlhogan @SMakaiTakori This is great - thank you for the changes!!

**Pull request merged**